src/packages/fetch-engine/src/getHash.ts

```diff
@@ -0,0 +1,17 @@
+import crypto from 'crypto'
+import fs from 'fs'
+
+export function getHash(filePath: string): Promise<string> {
```

Comment:
Could we use this here too https://github.com/prisma/prisma/blob/284891a1d04a1d98f358b5cd141c79c0106ed0c3/src/packages/fetch-engine/src/downloadZip.ts#L84

And then remove the hasha dependency?

Reply:
We still use hasha in `downloadZip`.

Reply:
Yes this is what I pointed out in my comment.
Could we use `getHash` everywhere instead?